pkgs/build-support/rust/import-cargo-lock.nix

```diff
@@ -133,13 +150,19 @@ let
       else if gitParts != null then
       let
         missingHash = throw ''
-          No hash was found while vendoring the git dependency ${pkg.name}-${pkg.version}. You can add
+          No hash was found while vendoring the git dependency ${pkg.name}-${pkg.version} or ${pkg.name}-${pkg.version}${suffix}. You can add
           a hash through the `outputHashes` argument of `importCargoLock`:
 
           outputHashes = {
             "${pkg.name}-${pkg.version}" = "<hash>";
           };
 
+          or
+
+          outputHashes = {
+            "${pkg.name}-${pkg.version}${suffix}" = "<hash>";
+          };
+
           If you use `buildRustPackage`, you can add this attribute to the `cargoLock`
           attribute set.
```

Comment:
To support duplicate outputHashes, it allows the commit hash to be added to the suffix.

Reply:
The format works with https://github.com/junjihashimoto/qdrant/blob/nix/v1.8.4/flake.nix .

Reply:
Hi @risicle, would you like to try this branch?